src/tokenizer/alphanum_only.rs

```diff
@@ -1,3 +1,28 @@
+//! # Example
+//! ```
+//! extern crate tantivy;
+//! use tantivy::tokenizer::*;
+//!
+//! # fn main() {
+//!
+//! let tokenizer = RawTokenizer
+//!   .filter(AlphaNumOnlyFilter);
+//!
+//! let mut stream = tokenizer.token_stream("hello there");
+//! // is none because the raw filter emits one token that
+//! // contains a space
+//! assert!(stream.next().is_none());
+//!
+//! let tokenizer = SimpleTokenizer
+//!   .filter(AlphaNumOnlyFilter);
+//!
+//! let mut stream = tokenizer.token_stream("hello there 💣");
```

Comment:
Awesome thank you!

Reply:
I'd like to do something like this for each one. Helps me to understand the code as well. :)